### PR TITLE
HZ-6 layout and add some folders

### PR DESCRIPTION
**Pull request by jinzhih**
finish HZ-6layout "email verification"
add theme file 
add folders for the structure
run "npm  run link" before push

### CODE REVIEW

File: src/components/EmailVerification/EmailVerification.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import * as React from "react";
+import Button from "@mui/material/Button";
+import CssBaseline from "@mui/material/CssBaseline";
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import Container from "@mui/material/Container";
+
+const EmailVerification = () => {
+  return (
+    <React.Fragment>
```
Comment (yuanningd) [inline src/components/EmailVerification/EmailVerification.tsx]: Use <> instead. 
image

File: src/pages/emailverification.tsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import type { NextPage } from "next";
+import EmailVerification from "../components/EmailVerification";
+
+const EmailVerificationPage: NextPage = () => {
+  return <EmailVerification />;
```
Comment (yuanningd) [inline src/pages/emailverification.tsx]: Why do you encapsulate the layout content into another component? To be honest, I didn't see the benefit of doing this. I think we don't need the EmailVerification component, you can just write the layout in this file

File: src/components/EmailVerification/EmailVerification.tsx (hunk 1 above)
Comment (jinzhih) [reply src/components/EmailVerification/EmailVerification.tsx]: Use <> instead.--done

File: src/pages/emailverification.tsx (hunk 2 above)
Comment (jinzhih) [reply src/pages/emailverification.tsx]: delete EmailVerification component, put the layout in the pages/emailverification.tsx . I checked Next documentation, change emailverification to email-verification to make the url readable

**Review by yuanningd — APPROVED** [review]

**Pull request merged**